config/config-example.js

```diff
@@ -92,6 +75,53 @@ Main's SSL deploy script from Let's Encrypt looks like:
  */
 exports.proxyip = false;
 
+// subprocesses - the number of child processes to use for various tasks
+//   It can be globally set to 0 for minimum hardware specs.
```

Comment:
```suggestion
// subprocesses - the number of child processes to use for various tasks.
//   Can be set to `0` instead of `{...}` to stop using subprocesses, if you're running out of RAM.
```